docs/docs/keymaps/behaviors/hold-tap.mdx

```diff
@@ -53,7 +53,9 @@ Note that you can also [define a custom hold-tap](#custom-hold-tap-examples) and
 
 ## Layer-Tap
 
-The "layer-tap" behavior works identically to the mod-tap behavior, but instead of outputting one of two keys, it activates a specified layer as its "hold" action.
+The "layer-tap" behavior works similarly to the mod-tap behavior, but instead of outputting one of two keys, it activates a specified layer as its "hold" action.
+
+By default, layer-tap is configured with the ["tap-preferred" `flavor`](#interrupt-flavors).
```

Comment:
mod-tap fully explains what this means; we could do something similar, but I felt like that might be belaboring it:

```suggestion
- If you hold the key for longer than 200ms, the layer specified as the first parameter is activated.
- If you tap the key (release before 200ms), the keycode specified as the second parameter is sent.

By default, layer-tap is configured with the ["tap-preferred" `flavor`](#interrupt-flavors).
```

Reply:
I think "works similarly to the mod-tap" covers it well enough.